FreeRTOS/Demo/CORTEX_MPU_M3_MPS2_QEMU_GCC/init/startup.c

```diff
@@ -189,6 +192,7 @@ void Debug_Handler( void )
         " mrsne r0, psp                                                      \n"
         " ldr r1, =vHandleMemoryFault                                        \n"
         " bx r1                                                              \n"
+        " .ltorg                                                             \n"
```

Comment:
Are you facing any problem because of which you are using these `ltorg`? If not, we should omit those and let the compiler create literal pools as needed.